src/libraries/System.Private.Xml/src/System/Xml/XmlEncoding.cs

```diff
@@ -277,7 +280,7 @@ public override int GetCharCount(byte[] bytes)
 
         public override Encoder GetEncoder()
         {
-            return null;
+            return null!;
```

Comment:
Depending on how pervasive this particular subtype is, it might indicate that the base class might need to be annotated differently.

Reply:
This case was weird - all callsites which used this method expected this to be not-null and this type is never used directly so we ended up never calling this specific implementation returning null - I've decided to suppress this since this type is not public.

Reply:
just double checked - those are only used internally for decoding (only when decoding is auto-detected)